src/sentry/replays/utils.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, Dict, Optional
+from urllib.parse import urlencode
+
+from django.conf import settings
+from django.http import StreamingHttpResponse
+from urllib3.response import HTTPResponse
+
+from sentry.net.http import connection_from_url
+from sentry.utils.profiling import RetrySkipTimeout
+
+_replays_pool = connection_from_url(
+    settings.SENTRY_REPLAYS_SERVICE_URL,
+    retries=RetrySkipTimeout(
+        total=3,
+        allowed_methods={"GET"},
+    ),
```

Comment:
This will report any retries with the key `profiling.client.retry`. You can either copy it here and change it or fix up the implementation in profiling to allow you to change the key. fwiw, the implementation in profiling is copied from the snuba connection pool.

Reply:
copied over!